help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -1420,12 +1420,35 @@
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
-      <h4>Logix</h4>
+      <h4 id="common_logix_action">Logix</h4>
 
-      <p>LogixNG has an action called Logix that emulates how JMRI Logix works. The action Logix
-      has one expression and a number of boolean actions. A boolean action is an action that takes
-      a boolean value (true/false) and then decides what to do. The action Logix was added mainly
-      to support import of JMRI Logix to LogixNG.</p>
+      <p>LogixNG has an action called <strong>Logix</strong> that emulates how JMRI Logix works. The
+      action Logix has one expression and a number of <a href="#boolean_logix_action">Logix Actions</a>.
+      A Logix Action responds to <strong>True</strong>, <strong>False</strong>, or
+      <strong>Either</strong> Logix evaluation states.  The action Logix was added mainly to support
+      import of JMRI Logix to LogixNG.</p>
```

Comment:
A suggestion is to make it more clear that the Logix action has only one Logix Action. To use several Logix Actions, use the Many action.